actors/runtime/proof/verify.go

```diff
@@ -23,6 +25,25 @@ import (
 //}
 type SealVerifyInfo = proof0.SealVerifyInfo
 
+type AggregateSealVerifyInfo struct {
+	Number                abi.SectorNumber
+	DealIDs               []abi.DealID
```

Comment:
@dignifiedquire This structure represents the info passed to proof verification. I'm curious as to why we need to pass deal ids here. Is this actually used, or are we plumbing useless data? AFAIK the proof verification routine can't use these to look up any information.

Reply:
@magik6k will have the best insight here, as we don't directly use deal information in the aggregation API

Reply:
No I'm pretty sure it's nothing to do with Lotus. This structure exists to encapsulate the data passed directly to proofs, via the ffi library. The non-aggregate version goes [directly to the VerifySeal FFI method](https://github.com/filecoin-project/filecoin-ffi/blob/master/proofs.go#L27). The data here was dictated to us by proofs team, though for dependency reasons the structure is defined in actors.

It looks like the deal IDs are indeed unused (not passed to Rust) and didn't need to be here, which we probably overlooked in a rush last year. It might not be worth the effort to remove from the non-aggregate version (which I hope we'll deprecate entirely soon), but @ZenGround0 let's not add them here until something forces us to.